mobile-app/lib/ui/views/learn/challenge/challenge_viewmodel.dart

```diff
@@ -657,17 +658,22 @@ class ChallengeViewModel extends BaseViewModel {
         },
       );
       if (testRes?.value['pass'] == null) {
-        log('TEST FAILED: ${test.instruction} - ${test.javaScript} - ${testRes?.value['error']}');
+        log('TEST FAILED: ${test.instruction} - ${test.javaScript} - ${testRes?.value['err']}');
         failedTest = failedTest ?? test;
+        failedTestErr = failedTestErr ?? testRes?.value['err'] as Map;
         failedTestsConsole.add(
-          '<li>${test.instruction}</li>',
+          '<li>${test.instruction.replaceAll('--fcc-expected--', (testRes?.value['err']['expected'] ?? '').toString()).replaceAll('--fcc-actual--', (testRes?.value['err']['actual'] ?? '').toString())}</li>',
```

Comment:
Can we create a simple function for this instead?

Reply:
Yea, you're right. I've updated the PR to add one